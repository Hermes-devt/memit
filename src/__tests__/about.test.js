
import React from 'react';
import {render, cleanup} from '@testing-library/react';
import '@testing-library/jest-dom';
import About from '../components/footer/about';

afterEach(cleanup);

it('expects it to render!', ()=>{
  const {asFragment} = render(<About/>);
  expect( asFragment() ).toMatchSnapshot();
})
