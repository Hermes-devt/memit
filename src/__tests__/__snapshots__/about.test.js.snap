// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expects it to render! 1`] = `
<DocumentFragment>
  <div>
    <h2
      style="color: orange;"
    >
      About
    </h2>
    <div>
      This is a application that lets you plugin your daily learnings to a set schedule so it allows you to repeat the information again just before you forget about it. Thereby not wasting your hard earned wisdom the the fragility of time.
    </div>
  </div>
</DocumentFragment>
`;
